electrode-ota-server-diregister/src/index.js

```diff
@@ -33,16 +33,16 @@ const result = (obj, dep) => {
 export const HANDLERS = {
     electrode: (server, dep, options) => result(server, dep)
 };
-const resolveable = (name) => {
+const resolveable = name => {
     const ret = {name};
-    ret.promise = new Promise(function (_resolve, _reject) {
+    ret.promise = new Promise(((_resolve, _reject) => {
```

Comment:
too many ()?

Reply:
eslint fixes. each time i work with a ticket, trying to fix them one file at time.

Reply:
there seems to be an extra ()

Reply:
removed